h5p-bildetema/src/components/SubHeader/SubHeader.module.scss

```diff
@@ -27,6 +27,8 @@
   flex-wrap: wrap;
   font-size: $font-size-18;
   row-gap: 1rem;
+  flex: 1;
+  justify-content: flex-end;
```

Comment:
The show written words toggle gets compressed at some screen widths when `flex: 1`. Im considering to move the print button, so for now I think we only need to add `flex-wrap: wrap;` to `.subHeaderThemes`  

<img width="437" alt="Skjermbilde 2022-10-26 kl  10 03 29" src="https://user-images.githubusercontent.com/35261194/197969894-80a80014-44ee-4911-b64f-a346e222375f.png">